source/dotnet/Test/AdaptiveCards.Templating.Test/TestTransform.cs

```diff
@@ -11231,7 +11240,7 @@ public void TestIntInExpression()
                     ""placeholder"": ""Enter Estimated Mileage"",
                     ""min"": 1,
                     ""max"": 10,
-                    ""value"": ""${car_type_1.average_mileage}"" 
+                    ""value"": ""=car_type_1.average_mileage"" 
```

Comment:
Can this create a backward compatibility issue? '=' is not formally introduced to the users. The users have been able to do the integer substitution without the new operator.